Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
Yes actually I think @alerque makes a good point here, agreed. But the `default` target already contains `all` and everything else though, so in a way the `default` target is already acting as some sort of 'all'. So I think it's actually fine the way it is.. and `make test` is what @alerque is looking for? @alerque could you confirm?